scipy/stats/stats.py

```diff
@@ -1846,17 +1879,43 @@ def relfreq(a, numbins=10, defaultreallimits=None, weights=None):
 
     Examples
     --------
+    >>> import matplotlib.pyplot as plt
     >>> from scipy import stats
-    >>> a = np.array([1, 4, 2, 1, 3, 1])
-    >>> relfreqs, lowlim, binsize, extrapoints = stats.relfreq(a, numbins=4)
-    >>> relfreqs
+    >>> a = np.array([2, 4, 1, 2, 3, 2])
+    >>> res = stats.relfreq(a, numbins=4)
+    >>> res.frequency
     array([ 0.5       ,  0.16666667,  0.16666667,  0.16666667])
```

Comment:
This is incorrect output, should be ``array([ 0.16666667,  0.5       ,  0.16666667,  0.16666667])``